env/env.go

```diff
@@ -1,334 +1,17 @@
 package env
 
 import (
-	"encoding/json"
-	"fmt"
-	"html/template"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"reflect"
-	"strings"
-
+	"github.com/davecgh/go-spew/spew"
```

Comment:
spew is temporary. this is TODO: to figure out good logger which can pretty print structures.